ilastik/applets/labeling/labelingGui.py

```diff
@@ -133,7 +133,8 @@ def __init__(self):
             # Slot that gives a list of label names
             self.labelNames = None # labelNames.value
 
-    def __init__(self, parentApplet, labelingSlots, topLevelOperatorView, drawerUiPath=None, rawInputSlot=None, crosshair=True):
+    def __init__(self, parentApplet, labelingSlots, topLevelOperatorView, drawerUiPath=None, rawInputSlot=None,
+                 crosshair=True, add_3d_widget=False):
```

Comment:
with `add_3d_widget` I have the same suggestion as in volumina - as th ewidget is added anyhow and only visibility is altered. `3d_widget_visible` ?

Reply:
yeah, that name makes more sense. I'll change the name here and in Volumina, so it's consistent everywhere. Thanks!

Reply:
changed the name to `is_3d_widget_visible`, cause var names cannot start with a number :) ✅